data/json/furniture.json

```diff
@@ -5998,7 +6030,7 @@
     "id": "f_dishwasher",
     "name": "dishwasher",
     "looks_like": "f_oven",
-    "description": "This metal box used to spray hot water and soap at dirty dishes to make them clean and to save people an unpleasant chore.  Now, with the power off and it sitting for a while, it is starting to smell a bit off.",
+    "description": "This metal box used to spray hot water and soap at dirty dishes to make them clean and to save people an unpleasant chore.  Now, with the power gone and it sitting for a while, it's starting to smell a bit off.",
```

Comment:
This is one of those lateral changes just for the sake of change.

Reply:
No, this was for flow. Reuses the word 'off' in two different contexts and sounds wrong to me.